src/test/java/dividing/DivideMethodTest.java

```diff
@@ -0,0 +1,34 @@
+package dividing;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DivideMethodTest {
+    private final DivideMethod divide = new DivideMethod();
+
+    @Test
+    void globalProgramTest() {
```

Comment:
Please follow our convention for test names `shouldDoSomethingIfSomenthing`
In this case it could be `shouldReturnFormattedDivisionResultIfValidParameters`